CHANGELOG/CHANGELOG-2.1.md

```diff
@@ -1,3 +1,56 @@
+# Release notes for release-2.1
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v2.1.0
+
+## Changes by Kind
+
+### Bug or Regression
+ - External-provisioner may have [stopped provisioning](https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/pull/103) for a PVC depending on certain timing conditions (provisioning failed once, next attempt currently running, PVC update arrives from API server). ([#564](https://github.com/kubernetes-csi/external-provisioner/pull/564), [@pohly](https://github.com/pohly))
+ - Producing storage capacity may have failed with `the object has been modified` errors ([#568](https://github.com/kubernetes-csi/external-provisioner/pull/568), [@pohly](https://github.com/pohly))
+ - `k8s.io/csi-translation-lib` dependency is updated from v0.20.0 to v0.20.4 to adopt fixes in the upstream library. ([#571](https://github.com/kubernetes-csi/external-provisioner/pull/571), [@ialidzhikov](https://github.com/ialidzhikov))
```

Comment:
sorry I just updated the release note for this PR since it's a obsolete now. Could you refresh again?

Reply:
Done.